{{cookiecutter.project_name}}/CONTRIBUTING.md

```diff
@@ -57,7 +57,7 @@ Ready to contribute? Here's how to set up **{{cookiecutter.project_name }}** for
 
 2. Clone your fork locally:
 
-        $ git clone git@{{ cookiecutter.vs_url }}:{% if cookiecutter.vs|lower == "bitbucket" %}7999/~{% endif %}your_name_here/{{ cookiecutter.project_name }}.git
+        $ git clone ssh://git@{{ cookiecutter.vs_url }}{% if cookiecutter.vs|lower == "bitbucket" %}:7999/~{% else %}/{% endif %}your_name_here/{{ cookiecutter.project_name }}.git
```

Comment:
Shouldn't the usage of ssh be left open? People could namely decide to use https instead also if they want.